manager-bundle/src/Resources/config/commands.yml

```diff
@@ -2,6 +2,12 @@ services:
     _defaults:
         autoconfigure: true
 
+    contao.command.initialize-application:
+        class: Contao\ManagerBundle\Command\InitializeApplicationCommand
+        arguments:
+            - '%kernel.project_dir%'
+            - '%contao.web_dir%'
```

Comment:
This is not correct. `%contao.web_dir%` is something different to the web dir that was used in the script handler before. It is supposed to be just `web` whereas `%contao.web_dir%` is the absolute path `/path/to/project/web` (`%kernel.project_dir%/web`).

The previous version of the script handler retrieved the "web" dir for the `assets:install` command etc. this way:

```php
    private static function getWebDir(Event $event): string
    {
        $extra = $event->getComposer()->getPackage()->getExtra();

        return $extra['symfony-web-dir'] ?? 'web';
    }
```

Reply:
good find! adjusted in 24077c048656f163db798b0883390ec80e122590

(I wonder why this only failed on windows though...)

Reply:
Making the path relative is not the right approach I think. You need to extract the web folder from the configuration as before.

Although it is kind of odd that there are two different configurations for the web dir currently. `contao.web_dir` in the bundle config and `extra.symfony-web-dir` in the `composer.json`.

Reply:
Mhh. I can't find a place where the `contao.web_dir` is set from the `extra.symfony-web-dir` section. If that's true, this an issue in itself. According to the core-bundle configuration it defaults to `%kernel.project_dir%/web` which would be wrong if you're using an adjusted config in your `composer.json`.

Reply:
I don't expect anyone to actually use `extra.symfony-web-dir` (or `contao.web_dir` for that matter) - otherwise this issue would have come up sooner, probably.

Also, in Symfony 4 it is `extry.symfony-public-dir` and in Symfony Flex it is `extra.public-dir` …

Reply:
see #2461